tsconfig.json

```diff
@@ -1,16 +1,19 @@
 {
 	"compilerOptions": {
-		"allowSyntheticDefaultImports": true,
 		"allowJs": true,
-		"declaration": true,
+		"allowSyntheticDefaultImports": true,
 		"esModuleInterop": true,
-		"jsx": "react", // use typescript to transpile jsx to js
-		"importHelpers": true,
-		"lib": ["dom", "es6"],
+		"forceConsistentCasingInFileNames": true,
+		"jsx": "react",
+		"isolatedModules": true,
+		"lib": ["dom", "dom.iterable", "esnext"],
+		"module": "esnext",
 		"moduleResolution": "node",
-		"outDir": "./assets/dist",
-		"target": "es6"
+		"noEmit": true,
+		"resolveJsonModule": true,
+		"skipLibCheck": true,
+		"strict": false,
+		"target": "es5"
```

Comment:
oh? not es6?

Reply:
that is setting from CRA, maybe they want to have high browser support, lemme bump that to es6 and see what happens

Reply:
updated to `es6`, `yarn dev` and `yarn prod` works as before

Reply:
p.s.: `yarn dev` and `yarn prod` is the same as `npm run dev` and `npm run prod`
want to avoid confusion